src/routes/Liquidity/Liquidity.js

```diff
@@ -2,13 +2,23 @@ import React from 'react';
 
 import {ChoiceItemType, MultiChoiceMenu} from '../../components/UI';
 import {useLiquidityProviders, useLiquidityTranslation} from '../../hooks';
-import {openInNewTab} from '../../utils';
+import {useL2Wallet} from '../../providers/WalletsProvider';
+import {openInNewTab, buildDynamicURL} from '../../utils';
 
 export const Liquidity = () => {
+  const {account: accountL2} = useL2Wallet();
```

Comment:
Why do we need this renaming? can we call it just `account`? also in the config file? and if so renaming, it will be better to rename it to `walletAddress` as the QS is named by banxa.

Reply:
I did it for clarity, `account` can be both for L1/L2 wallets.